include/openthread/instance.h

```diff
@@ -218,6 +218,14 @@ void otInstanceFactoryReset(otInstance *aInstance);
  */
 otError otInstanceErasePersistentInfo(otInstance *aInstance);
 
+/**
+ * This function gets the Thread protocol version.
+ *
+ * @returns the Thread protocol version.
+ *
+ */
+uint16_t otGetThreadVersion(void);
```

Comment:
What about moving this to [include/openthread/thread.h](https://github.com/openthread/openthread/blob/master/include/openthread/thread.h) and be renamed to `otThreadGetVersion()`?

Reply:
done.